packages/bot/src/stores/flyout-store.js

```diff
@@ -160,8 +160,9 @@ export default class FlyoutStore {
         const toolbox         = Blockly.derivWorkspace.toolbox_; // eslint-disable-line
         const is_flyout_click = event.path.some(el => el.classList && el.classList.contains('flyout'));
         const isToolboxClick  = () => toolbox.HtmlDiv.contains(event.target);
+        const isSearchInput   = () => document.querySelector('input[name="search"]') === event.target;
```

Comment:
@sam-binary can you just add some `onclick` event to the component and set some flags? and please use snake-case for naming

Reply:
great idea.